cms/models.py

```diff
@@ -97,6 +97,30 @@ class CategorizedFaqsPage(Page):
     parent_page_types = ['FaqsPage']
 
 
+class GenericPage(Page):
+    """
+    CMS Page for custom tabs on the program page
```

Comment:
if this is an accurate description of the class, i think `ProgramTab` or `ProgramTabPage` would be a better/more accurate name than `GenericPage`